src/pyfaf/actions/pull_reports.py

```diff
@@ -140,6 +138,8 @@ def _get_report(self, name):
     def run(self, cmdline, db):
         if cmdline.master is not None:
             self.master = cmdline.master
+        # can come from cmdline or from config
+        if self.master:
             self._load_known()
```

Comment:
We just need to move line 143 one level up, as we want to load known on each run.